packages/plugins/urlWriteback/mod.ts

```diff
@@ -25,16 +25,27 @@ interface WritebackPayload {
  * as a handler for the publishing system onUrlChange event in order to
  * save published urls back into the Knowledge Graph.
  *
+ * The fieldToFeatureMap is an optional parameter that can be used to define
+ * a custom field to store other features that are powered by entity data
+ * i.e., custom review forms that are powered by entity data but have a different
+ * CF to store the prod URL of the page.
+ * 
  * @public
  */
-export default function urlWriteback(config: UrlWritebackConfig) {
+export default function urlWriteback(config: UrlWritebackConfig, fieldToFeatureMap: Record<string, string> = {}) {
```

Comment:
think it would make sense for the map to be an optional property on the config, rather than a new second argument

Reply:
gotcha! I wasn't sure if I could add an option property on the config, but definitely makes more sense to have it there